src/types/table.ts

```diff
@@ -76,3 +98,141 @@ export interface TablePaginationAttributes {
 }
 
 export type RowBulkAction = boolean | { enabled: boolean, disabledTooltip?: string }
+
+export interface SortHandlerFunctionParam {
+  /** The key of the column to be sorted */
+  key: string
+  /** The key of the column previously sorted */
+  prevKey: string
+  /** The order in which to sort the column (asc or desc) */
+  sortColumnOrder: 'asc' | 'desc'
+  /** The data returned from the fetcher function response */
+  data: Array<Record<string, any>>
+}
+
+export interface CellAttrsParam {
+  headerKey: string
+  row: Record<string, any>
+  rowIndex: number
+  colIndex: number
+}
+
+export interface TableProps {
```

Comment:
nit: since we have two tables, could we rename this to `TablePropsShared`?

Reply:
Done